src/puppetlabs/trapperkeeper/services/status/status_service.clj

```diff
@@ -32,3 +32,16 @@
     (log/infof "Registering status callback function for %s service" service-name)
     (core/update-status-context (:status-fns (service-context this))
       service-name service-version status-version status-fn)))
+
+(defservice status-proxy-service
```

Comment:
Is this duplicated intentionally from `status_proxy_service.clj`?  This doesn't seem to be needed.